### DISCUSSION TITLE
Replace zlib with zlib-ng

### DISCUSSION THREAD

**Discussion by netcorefan1** [microsoft/vcpkg#30743] [Q&A] [answered]
Hello,
I would like to replace zlib with [zlib-ng](https://github.com/zlib-ng/zlib-ng/pull/1335). It seems that it is compatible with the old api if compiled with ZLIB_COMPAT (off by default).  Once I manage to get it installed in this way, what is the correct way to change a package dependency from zlib to zlib-ng? Is this a safe operation?

**Comment by dg0yt** [comment, marked as answer]
How: You make an empty overlay port for zlib which depends on zlib-ng with that feature.

**Reply by netcorefan1** [reply]
Many thanks, friend! The right direction on where to start looking for was all I needed!
